api/firstbook.py

```diff
@@ -94,12 +94,13 @@ def remote_pin_test(self, barcode, pin):
                 str(e),
                 self.NAME
             )
+        content = response.content.decode("utf8")
```

Comment:
Its really an upstream issue, so I'm fine merging as is, but I think we should really be using `response.text` here instead of just assuming `utf8`.

https://docs.python-requests.org/en/master/user/quickstart/#response-content